### PR TITLE
The program counts the number of rows in a file

### PR DESCRIPTION
**Pull request by Sergiy-1998**
Write a program to count the number of lines in a text file.

### CODE REVIEW

**Comment by python-teacher** [comment]
Please change the name of this pull request.
It should be self-descriptive. It should be clear from the name what is this pull request about.

File: hw_7.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"""
+Write a program to count the number of lines in a text file.
+"""
+
+
+def count_rows():
+    with open('file/file.txt', 'r') as f:
+        count = 0
+        for x in f.readlines():
```
Comment (python-teacher) [inline hw_7.py]: It's too complicated. There is an easier solution.

**Review by python-teacher — CHANGES_REQUESTED** [review]

File: hw_7.py (hunk 1 above)
Comment (python-teacher) [reply hw_7.py]: Also, we don't need this additional iteration through all these lines.

**Review by python-teacher — CHANGES_REQUESTED** [review]

File: hw_7.py (hunk 1 above)
Comment (Sergiy-1998) [reply hw_7.py]: done

File: hw_7.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"""
+Write a program to count the number of lines in a text file.
+"""
+
+
+def count_rows():
+    with open('documentation file/short documentation.txt', 'r') as f:
+        return sum(1 for x in f)
```
Comment (python-teacher) [inline hw_7.py]: It's not clear what happened here.

Comment (Sergiy-1998) [reply hw_7.py]: the loop goes through each line and writes each line as "1" to the list, after which everything is added

Comment (UkrainianEngineer) [reply hw_7.py]: It's not clear from this implementation.
It might be done easier.

Comment (Sergiy-1998) [reply hw_7.py]: done

File: hw_7.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+"""
+Write a program to count the number of lines in a text file.
+"""
+
+
+def count_rows():
+    return len(open('file/short documentation.txt', 'r').readlines())
```
Comment (python-teacher) [inline hw_7.py]: Too much of operation inside one operator.

Comment (Sergiy-1998) [reply hw_7.py]: done

File: hw_7.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+"""
+Write a program to count the number of lines in a text file.
+"""
+
+
+def count_rows():
+    with open('file/short documentation.txt', 'r') as f:
+        line = f.readlines()
```
Comment (python-teacher) [inline hw_7.py]: It's a lot of lines, not just a single line.

Comment (Sergiy-1998) [reply hw_7.py]: done,renamed

File: hw_7.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+"""
```
Comment (python-teacher) [inline hw_7.py]: Please fix a module name.

Comment (Sergiy-1998) [reply hw_7.py]: done

Comment (python-teacher) [reply hw_7.py]: `Count lines file`?

It should be changed to `count_file_lines` or `calculate_lines_of_file`, etc.

**Review by python-teacher — APPROVED** [review]

**Pull request merged**